utils/insert_data_models_db_utils.py

```diff
@@ -0,0 +1,57 @@
+import os
+from hafifa.data_base import data_models
+from hafifa.data_base.SQLAlchemy import SQLAlchemyHandler
+from hafifa.utils import frame_utils, metadata_utils
+
+
```

Comment:
Why not creating a class for managing those functions instead of having them around ?

Try staying consistent with your functions. some of them getting the db and some are not.

Why not changing the SQLAlchemyHandler to a sort of db handler or creating new class for handling db transactions that connected to your project ?

or if u decide to go with functional programming, have a script of db insert for each type of model.